tests/main/snap-discard-ns/task.yaml

```diff
@@ -7,31 +7,30 @@ details: |
 prepare: |
     . $TESTSLIB/snaps.sh
     install_local test-snapd-tools
-environment:
-    # TODO: vary by distro later
-    LIBEXECDIR: /usr/lib/snapd/
 execute: |
+    . "$TESTSLIB/dirs.sh"
```

Comment:
Nice, thank you!

I was thinking we should add `spread.sh` that is automatically sourced by each script implicitly. This would allow us to reduce the boilerplate.